proto/rill/runtime/v1/api.proto

```diff
@@ -504,6 +504,8 @@ message UnpackEmptyResponse {}
 message GenerateMetricsViewFileRequest {
   string instance_id = 1 [(validate.rules).string = {pattern: "^[_\\-a-zA-Z0-9]+$"}];
   string connector = 2;
+  string database = 6;
+  string db_schema = 7;
   string table = 3;
```

Comment:
The terminology of `database` and `db_schema` feels off – it seems weird to type out `database` in one case and use a `db` abbreviation in another. What about just `database`, `schema`, and `table`? (It's annoying that `schema` can have multiple meanings, but that's a problem in so many tools that people are hopefully used to it by now. If we really want to disambiguate, an option would be `database_schema`.)

Reply:
I actually kept it just `schema` earlier but later changed it since some of the API responses already had a schema field and thought to keep same across request and response. `database_schema` felt too verbose but I can keep it to reduce the weirdness.